src/pages/uxp/reference-js/Global Members/HTML DOM/NodeList.md

```diff
@@ -1,13 +1,23 @@
----
-jsDoc: true
----
 
 <a name="nodelist" id="nodelist"></a>
 
 # window.NodeList
 **See**: https://developer.mozilla.org/en-US/docs/Web/API/NodeList  
 
 
+<a name="new-nodelist-new" id="new-nodelist-new"></a>
+
+## NodeList(staticList, updater)
+Creates an instance of NodeList.
+
+
+| Param | Type |
+| --- | --- |
+| staticList | `*` | 
+| updater | `*` | 
```

Comment:
I can't say I'm super happy with this parameter definition, but there isn't a lot we can do about that given that such a constructor, according to MDN, isn't usually existant in browser environments (probably there isn't a real use-case for a 3P dev to use that constructor, anyways).

Reply:
🟢